Changes

```diff
@@ -6155,7 +6155,7 @@ Bug Fixes:
 - #5575: Random states are not marshallable across architectures
 - #5579: camlp4: when a plugin is loaded in the toplevel,
   Token.Filter.define_filter has no effect before the first syntax error
-- #5585: typo: "explicitely"
```

Comment:
This one looks intentional.

Reply:
Yes, I know.
But if codespell should eventually be used automatically it should not stop on this case. "Correcting" this word is simpler, right?

Reply:
No, it means it can’t be done until there’s a way to make exceptions. The tool is supposed to serve the task!

Reply:
One can just declare that the bug fixes change listing should list the correct spelling and not the one that was fixed. Problem solved! :)

Reply:
No, like a stuck record! There will still be a point in the future where a misspelling is not a misspelling.